src/routes/NewBoard/NewBoard.tsx

```diff
@@ -8,20 +8,28 @@ import Parse from "parse";
 import {API} from "api";
 import {getColorForIndex} from "constants/colors";
 
-export type NewBoardProps = RouteComponentProps;
-
-function NewBoard(props: NewBoardProps) {
+function NewBoard(props: RouteComponentProps) {
   const [name, setName] = React.useState(getRandomName());
   function handleChangeName(e: any) {
     const name = (e.target as HTMLInputElement).value;
     setName(name);
   }
 
-  async function onLogin() {
+  async function onAnonymousLogin() {
     await AuthenticationManager.signInAnonymously(name);
     await onCreateBoard();
   }
 
+  async function onLogout() {
+    await Parse.User.logOut();
+    props.history.push("/");
```

Comment:
I have to redirect, since a logout does not change the state and therefore won't re-render everything.
We could start pushing actions for logins/logouts on redux and extract the calls to the middleware to fix this. If you would prefer it that I would suggest to open an issue for the refactor :)

Reply:
Hm in my opinion that actually sounds like a good idea
We could create AuthActions, -Reducer and -Middleware for Login, Logout etc.